src/screens/PostScreen.tsx

```diff
@@ -1,43 +1,87 @@
-      <Text>
-        This organization is doing amazing work tackling the complex root causes
-        of the issue.
-      </Text>
-      <ProfilePlaceholder />
-      <Text>vppraggie</Text>
-      <Text>September 21</Text>
-      <Text>Thanks for sharing!</Text>
-    </View>
+    <ScrollView contentContainerStyle={styles.container}> 
+    {/* Instead of using View, use ScrollView to ensure it can move. Then, instead of style={}, we use contentContainerStylenpx */}
+      <View style={styles.name}>
+        <View style={styles.imageName}>
+          <ProfilePlaceholder />
+          <View style={styles.userName}>
+            <Text style={styles.userNameFont}>rbeggs</Text>
+          </View>
+        </View>
+        <View style={styles.date}>
+          <Text>September 19</Text>
+        </View>
+      </View>
+
+      <View style={styles.text}>
+        <Text style={styles.textWords}>
+          In response to the growing homelessness crisis in San Francisco, a
+          local nonprofit organization, Code Tenderloin, has launched a
+          comprehensive initiative aimed at providing long-term solutions for
+          individuals experiencing homelessness. The organization, founded in
+          2015, is dedicated to addressing both immediate needs and underlying
+          causes of homelessness through a combination of shelter services, job
+          training programs, and mental health support. read more online:
+          https://www.codetenderloin.org/
+        </Text>
+      </View>
+
+      <Image
+        source={{
+          uri: 'https://cdn.britannica.com/51/178051-050-3B786A55/San-Francisco.jpg',
+        }}
+        style={styles.image}
+      />
+
+      <View style={styles.responses}>
+        <View style={styles.likeCounter}>
+          <HeartIcon />
+          <Text style={styles.likeCounterNumerical}>256 Likes</Text>
+        </View>
+        <ShareIcon />
+      </View>
+
+      <View style={styles.line}></View>
+
+      <View style={styles.name}>
+        <View style={styles.imageName}>
+          <ProfilePlaceholder />
+          <View style={styles.userName}>
+            <Text style={styles.userNameFont}>phillip_ye</Text>
+          </View>
+        </View>
+        <View style={styles.date}>
+          <Text style={styles.textWords}>September 20</Text>
+        </View>
+      </View>
+
+      <View style={styles.commentText}>
+        <Text style={styles.textWords}>
+          This organization is doing amazing work tackling the complex root
+          causes of the issue.
+        </Text>
+      </View>
+      <View style={styles.name}>
+        <View style={styles.imageName}>
+          <ProfilePlaceholder />
+          <View style={styles.userName}>
+            <Text style={styles.userNameFont}>vppraggie</Text>
+          </View>
+        </View>
+        <View style={styles.date}>
+          <Text>September 21</Text>
+        </View>
+      </View>
+      <View style={styles.commentText}>
```

Comment:
Same here, I would name this to commentContainer. Generally, you would use the word text when the style should be applied to `<Text>`, and container when it should be applied to `<View>`. It just makes more sense that commentText would be applied to some text which makes it easier for other people to read your code.